resources/js/app.js

```diff
@@ -8,8 +8,6 @@ import Routes from './routes';
 import Alert from './components/Alert.vue';
 import SchemeToggler from './components/SchemeToggler.vue';
 
-import.meta.glob(['../img/**']);
-
```

Comment:
This was here for the Favicon. We don't need it anymore as we inline the favicon.